app/views/pages/index.html.erb

```diff
@@ -10,3 +10,18 @@
     <%= link_to "Subscribe to newsletter", new_subscription_path %>
   </nav>
 <% end %>
+
+
+<%= form_tag do %>
+  <%= label_tag :email %>
+  <%= text_field_tag :email, '',
+        inputmode: :email,
+        list: :emails,
+        'data-remote': true,
+        'data-method': :get,
+        'data-url': '/emails',
+        'oninput': 'jQuery(this).change();'
```

Comment:
`data-remote`, `data-method`, and `data-url` options are provided by the [jquery-ujs](https://github.com/rails/jquery-ujs/wiki/ajax) library (which is probably already available in your Rails app). 

Those `data-` options, in conjunction with the `oninput` event handler will trigger an AJAX request to `/emails` each time the input field is typed in.